packages/shared/src/components/ShareOptionsMenu.tsx

```diff
@@ -66,7 +66,7 @@ export default function ShareOptionsMenu({
     {
       icon: <MenuIcon Icon={ShareIcon} />,
       text: 'Share article via...',
-      action: onShare,
+      action: () => onShare(post),
```

Comment:
Did onShare not already always take the post?

Reply:
It was not. From the type it came from, it was empty, but it probably worked as the post came from the parent component where it was triggered and handled. Now that we are reusing the type, it is better to be explicit with the function than handling it from the parent's implementation.

Reply:
Oh good find then! Lucky this didn't break randomly